tools/webshell/webshell.cc

```
@@ -0,0 +1,2184 @@
+    explicit resource_handler(request_control& request_control, const config& cfg, bool is_https)
+        : gated_handler("resource", request_control, is_https)
+        , _config(cfg)
+    {}
+
+    future<> load_manifest() {
+        if (_config.webshell_resource_manifest_path.empty()) {
+            _resource_manifest = resources::webshell_resources_manifest;
+        } else {
+            try {
+                _resource_manifest_storage = co_await resources::load_resource_manifest(_config.webshell_resource_manifest_path);
+                _resource_manifest = std::span(_resource_manifest_storage.data(), _resource_manifest_storage.size());
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)